index.html

```diff
@@ -37,21 +42,24 @@ <h1 class="h">I'm Uthman <br> Glad to see <br>you!</h1>
             </div>
         </div>
     </section>
-    <section class="section2">
-        <div>
-            <section class="border-box">
-                <img src="images/Snapshoot Portfolio.png" class="snapshot">
+    <section class="section">
+            <section class="border-box sec">
+                <div class="sec-left">
+                <img src="images/Snapshoot Portfolio.png" class="status">
+                <img src="images/section 2 desk.png" class="statusd">
+                </div>
+                <div class="sec-right">
```

Comment:
- [ ] Kindly add the technologies badges and the `see project` button for the 2nd project cards 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/206857343-b3b66127-f6be-42f0-9b51-3ebeff594c48.png)

## Figma design:
![Uploading image.png…]()